specification/gedcom.md

```diff
@@ -402,16 +402,67 @@ as described in [Extensions].
-- An **extended-use standard structure** is a structure whose type, tag, and meaning are defined in this document and whose superstructure is a tagged extension structure.
+This specification also provides multiple ways to for extension authors to go beyond the specification without submitting a feature request, which are described in the remainder of this section.
 
-Extension structures may have substructures, which may be either tagged extension structures of extended-use standard structures.
+A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure. Their meaning is defined by their tag, as is discussed more fully in the section [Extension Tags].
+
+Any substructure of a tagged extension structure that uses a tag matching `stdTag` is an **extension-defined substructure**.
+Substructures of an extension-defined substructure that uses a tag matching `stdTag` are also extension-defined substructures.
+The meaning and use of each extension-defined substructure is defined by the tagged extension structure it occurs within, not by its tag alone nor by this specification.
+
+:::example
+In the following
+
+```gedcom
+0 @P1@ _LOC
+1 NAME Βυζάντιον
+2 DATE FROM 667 BCE TO 324
+1 _POP 15149358
+2 DATE 31 DEC 2020
+0 @I1@ INDI
+1 BIRT
+2 _LOC @P1@
+```
+
+- Both uses of `_LOC` are tagged extension structures, as is `_POP`.
+- `_LOC`.`NAME` and `_LOC`.`NAME`.`DATE` are both extension-defined substructures. Their meaning is defined by the specification defining `_LOC`.
+- `_POP`.`DATE` is an extension-defined substructure. Its meaning is defined by the specification defining `_POP`.
+- Even though both `DATE`s appear to have `g7:type-DATE` payloads, we can't know that is the intended datatype without consulting the defining specifications of `_LOC` and `_POP`, respectively. The first might be a `g7:type-DATE#period` and the second a `g7:type-DATE#exact`, for example.
+:::
+
+If an extension-defined substructure has a tag that is also used by one or more standard structures, its meaning and payload type should match at least one of those standard structure types.
+
+:::example
+An extension-defined substructure with tag "`DATE`" should provide a date or date period relevant to its superstructure, as do all `DATE`-tagged structures in this specification. Extensions should not use "`DATE`" to tag a structure describing anything else (even something that might reasonably be abbreviated "date", such as someone an individual dated).
+:::
+
+As a special case, a tagged extension structure can be defined to have a standard structure type.
+These are called **relocated standard structures** and can only appear with superstructures that are not documented as a superstructure of that structure type in this specification.
```

Comment:
Is there a technical reason for this limitation?
This might prevent arguably legitimate cases, such as DATE.TIME with local time, and DATE._TIME with UTC (say because an obit stated time of death, but the time zone boundary or DST rule was updated in a later year so listing the UTC disambiguates which time zone it was in at the time).   With this one would have to define _TIME to use something other than g7:type-Time even if the syntax is the same as g7:type-Time.   I just made up this use case right now and I don't need it, there may be other legit ones that people do need.

Reply:
`g7:type-Time` is a payload type, not a structure type, and its not limited by this clause. It's perfectly OK to define a structure type _UTCTIME with payload type g7:type-Time and use it next to a g7:TIME structure type. But `g7:TIME` is a structure type and is limited by this clause.

Why the limit?
- `g7:TIME` should be a full description of the structure type. It shouldn't have variants for `TIME` vs `_TIME` as the tag.
- Structure types define semantics, and singular structures define semantics that don't generalize to plurality. Is a DATE with two TIMEs suppose to be an event that happened twice in a day, or under uncertainty about the time, or the same time in two time zones, or ...?

Result: if there's already a g7:TIME substructure, you can't define a new tag for it. Either it's already plural, in which case use several with tag TIME, or you have some new plural semantics in mind and should define a structure type to explain that.